### DISCUSSION TITLE
Github page doesn't work after making it private and public again

### DISCUSSION THREAD

**Discussion by joselops** [community/community#37794] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I tried to change the repository to private and, when I made it public again, the page doesn't load anymore. Does anyone know how to fix it? I already tried deleting the repository and creating it again, but it won't work.

**Comment by RubenGamarrarodriguez-tomtom** [comment, marked as answer]
It make take some time to be indexed again so that it can be accessed

**Reply by joselops** [reply]
Thanks!

**Comment by rajisgreat** [comment]
You need to publish it again the same way you did before.
